src/ChatCore.ts

```diff
@@ -8,6 +8,16 @@ import { ApiMessageRequest } from "./models/endpoints/MessageRequest";
 import { ApiResponse } from "./models/http/ApiResponse";
 import { QueryParams } from "./models/http/params";
 import { ApiResponseValidator } from "./validation/ApiResponseValidator";
+import { RawResponse } from "./models";
+
+interface Poster {
```

Comment:
remove this interface, since we decided to postpone exposing this. And refer the change in line 29 to be `private httpService: HttpService;`

Reply:
talked offline, opting not to do this because the interface helps decouple chatcore from the implementation